### PR TITLE
better form handling

### PR DESCRIPTION
**Pull request by Sebastian-Paterson-Jones**
better form handling

### CODE REVIEW

**Review by James-Burgess — COMMENTED** [review]
gonnaa get a preview deploy up now and look but looking v coool!

File: public/index.html [hunk 1]
```diff
@@ -142,6 +139,7 @@ <h1>Contact Me</h1>
 <script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/3.2.0/anime.js"></script>
 <script src="/assets/scripts/typewriter.js"></script>
 <script src="/assets/scripts/vendors/rough.js"></script>
+<script src="anime.min.js"></script>
```
[the remark this answers is not in the document]
Comment (Sebastian-Paterson-Jones) [reply public/index.html:142]: yes

[the remark this answers is not in the document]
Comment (James-Burgess) [reply public/index.html:142]: sweet

**Review by James-Burgess — APPROVED** [review]
Had a pull and added an update that flips back after sending. 
Would be cool to see one of you spinners in the button at some point if you are up to it x

**Pull request merged**